packages/caver-rtm/src/index.d.ts

```diff
@@ -0,0 +1,269 @@
+/*
+    Copyright 2021 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { ContractOptions, EventData } from '../../caver-contract/src'
+import { AccountKeyObject } from '../../caver-core-helpers/src/formatters'
+import AbstractTransaction from '../../caver-transaction/src/transactionTypes/abstractTransaction'
+import ValueTransfer from '../../caver-transaction/src/transactionTypes/valueTransfer/valueTransfer'
+import { SignatureObject } from '../../caver-utils/src/utils'
+
+export interface AccountObject {
+    accType: number
+    account: {
+        balance: string
+        humanReadable: boolean
+        key: AccountKeyObject
+    }
+}
+
+export interface BlockObject {
+    blockscore: string
+    extraData: string
+    gasUsed: string
+    governanceData: string
+    hash: string
+    logsBloom: string
+    number: string
+    parentHash: string
+    receiptsRoot: string
+    reward: string
+    size: string
+    stateRoot: string
+    timestamp: string
+    timestampFoS: string
+    totalBlockScore: string
+    transactions: TransactionObject[]
+    transactionsRoot: string
+    voteData: string
+}
+
+export interface BlockWithConsensusInfoObject extends BlockObject {
+    committee: string[]
+    originProposer: string
+    proposer: string
+    round: number
+}
+
+export interface TransactionObject {
```

Comment:
```suggestion
export interface Transaction {
```

Reply:
It is duplicated name that type `Transaction` at `caver-transaction/index.d.ts`
I think hard to mange later. is it okay?

Reply:
Um... that's true. Then how about `TransactionFromNode`?

Reply:
Fixed it. please check c76ca77